src/models/TrayOrder.ts

```diff
@@ -45,14 +45,18 @@ export class TrayOrder {
     const columnContainingB = this.order.findIndex(
       column => column.indexOf(b) > -1
     );
-    if (columnContainingA < columnContainingB) {
```

Comment:
Fix a bug in this method -- it was previously sorting Top-to-Bottom then Right-to-Left.  It should actually sort by Right-to-Left first, then Top-to-Bottom.